RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -700,6 +700,14 @@ void finalizeMaximize(boolean speculative)
 		// blocks first hit, but doesn't stack with blood bubble
 		addBonusToMaximize($item[Eight Days a Week Pill Keeper], 100);
 	}
+	if (possessEquipment($item[June cleaver])) {
+		addBonusToMaximize($item[June cleaver], 100); // We want to ramp this up and the NCs are nice as well
+	}
+	if (possessEquipment($item[designer sweatpants])) {
+		if (get_property("sweat").to_int() < 90 || get_property("_sweatOutSomeBoozeUsed").to_int() < 3) {
```

Comment:
If ya like the change to use all 3 sweat outs at once (see below), then I'd say delete the second check here. Otherwise it would be overcharging often. 

Actually I think it should get deleted anyways because it will be over halfway through the day when we start drinking and up until then we will be overcharging sweat. Liver cleanse won't get used since no booze has been drunk